From 3051da84c53dc09a58bbba3b637a5fc1cf7f7ccf Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Tue, 30 Sep 2014 16:40:54 +0200
Subject: [PATCH 1/2] Remove the shebang from the completion file

It's useless as completion file is sources and it makes rpmlint sad.
---
 bash_completion.d/minipro | 2 --
 1 file changed, 2 deletions(-)

diff --git a/bash_completion.d/minipro b/bash_completion.d/minipro
index bd40d6c..8bd620b 100644
--- a/bash_completion.d/minipro
+++ b/bash_completion.d/minipro
@@ -1,5 +1,3 @@
-#!/bin/bash
-
 _minipro () {
 	COMPREPLY=()
 	local cur="${COMP_WORDS[COMP_CWORD]}"
-- 
1.9.3

